version.h.in

```diff
@@ -13,4 +13,6 @@
 #define KERNEL_PATCHLEVEL      @KERNEL_PATCHLEVEL@
 #define KERNEL_VERSION_STRING  @KERNEL_VERSION_STRING@
 
+#define BUILD_VERSION          @BUILD_VERSION@
```

Comment:
The comment at the top is no longer true.

Reply:
comment updated to be more precise.